include/cpr/session.h

```diff
@@ -66,6 +66,7 @@ class Session : public std::enable_shared_from_this<Session> {
     void SetParameters(Parameters&& parameters);
     void SetHeader(const Header& header);
     void UpdateHeader(const Header& header);
+    Header& GetHeader();
```

Comment:
```suggestion
    Header& GetHeader();
    [[nodiscard]] const Header& GetHeader() const;
```

Could you please also add a const version of this function?